docs/source/_static/js/main.js

```diff
@@ -0,0 +1,7 @@
+$(document).ready( function () {
```

Comment:
This is jQuery, a slightly big JS lib. Which I feel we can avoid with plain simple vanilla JS. Using DOM manipulation.